dokka-subprojects/plugin-javadoc/src/test/kotlin/org/jetbrains/dokka/javadoc/JavadocAccessorNamingTest.kt

```diff
@@ -80,7 +80,7 @@ internal class JavadocAccessorNamingTest : AbstractJavadocTemplateMapTest() {
 
                 assertEquals(setOf(
                     "issuesFetched",
-                    "isFoo()",
+                    "isFoo",
```

Comment:
Before the fix, it was wrong because, in this case, the getter and the backing field have the same name `isFoo`.

Reply:
No, this test is about the Javadoc format.
To fix #3351, `JavadocExternalLocationProvider.kt` should be fixed as well. 

I updated the PR description to "Introducing `isProperty` could also help to fix https://github.com/Kotlin/dokka/issues/3351"